### PR TITLE
Add small prop to button component

### PR DESCRIPTION
**Pull request by liondedan**
Button support small prop

### CODE REVIEW

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/67
Branch:
https://sofarsounds.github.io/maestro/branches/button-sizes
Storybook:
https://sofarsounds.github.io/maestro/32fd9b233c897fc350f3ccdbad1a95265018794d/
Build:
https://sofarsounds.github.io/maestro/32fd9b233c897fc350f3ccdbad1a95265018794d/dist.zip

File: packages/uikit/src/atoms/Button/index.tsx [hunk 1]
```diff
@@ -47,6 +48,12 @@ export const BaseButton = styled.button<ButtonProps>`
         pointer-events: none;
         cursor: disabled;
       `}
+
+    ${small &&
+      css`
+        height: 40px;
```
Comment (davidfloegel) [inline packages/uikit/src/atoms/Button/index.tsx]: Can I be a _massive_ pain in the ass and say that we should define dimensions in the theme?
we've got the default buttonHeight in there so maybe something like this?
```js
dimensions: {
  button: {
    small: {
     height: '40px',
     padding: '0 15px'
    },
   default: {
     height: '44px',
     padding: '0 25px'
   }
  }
}
```

Comment (liondedan) [reply packages/uikit/src/atoms/Button/index.tsx]: 👍

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/67
Branch:
https://sofarsounds.github.io/maestro/branches/button-sizes
Storybook:
https://sofarsounds.github.io/maestro/bb9fd424ae256d8bd358b234aa3e87831c51e127/
Build:
https://sofarsounds.github.io/maestro/bb9fd424ae256d8bd358b234aa3e87831c51e127/dist.zip

**Pull request merged**